lib/matplotlib/offsetbox.py

```diff
@@ -1,17 +1,26 @@
-"""
-The `.OffsetBox` is a simple container artist. Its child artists are
-meant to be drawn at a relative position to OffsetBox.  The [VH]Packer,
-DrawingArea and TextArea are derived from the OffsetBox.
-
-The [VH]Packer automatically adjust the relative positions of their
-children, which should be instances of the OffsetBox. This is used to
-align similar artists together, e.g., in legend.
-
-The DrawingArea can contain any Artist as a child. The
-DrawingArea has a fixed width and height. The position of children
-relative to the parent is fixed.  The TextArea is contains a single
-Text instance. The width and height of the TextArea instance is the
-width and height of the its child text.
+r"""
+Container classes for `.Artist`\s.
+
+`OffsetBox`
+    The base of all container artists.
```

Comment:
... defined in this module (because there's also Containers in the sense of matplotlib.container, which are unrelated...)